infer/resource.go

```diff
@@ -1248,6 +1248,7 @@ func (rc *derivedResourceController[R, I, O]) Delete(ctx context.Context, req p.
 		if err != nil {
 			return err
 		}
+
```

Comment:
Can we not adjust whitespace in files we don't otherwise touch?
```suggestion
```

Reply:
I had touched and apparently left this artifact when I reversed my changes.